src/components/MDXComponents/MDXLink.tsx

```diff
@@ -3,15 +3,20 @@ import ExternalLink from '@/components/ExternalLink';
 import { useCurrentPlatform } from '@/utils/useCurrentPlatform';
 
 export const MDXLink = (props) => {
-  const { href, children } = props;
+  let { href, children, hash } = props;
```

Comment:
Can we make a post-ga task to make this a const again? WE can use ternaries to get the same logic without mutating href